### DISCUSSION TITLE
AuthN and AuthZ support for REST API on Spring Boot/Quarkus Runtime

### DISCUSSION THREAD

**Discussion by durgapmenonmck** [apache/camel-karavan#668] [Q&A] [answered]
Hi Team,

Currently the support for configuring Security for REST API is limited in Karavan. 

We would need to include AuthN and AuthZ capabilities in the REST services created (Spring Boot/Quarkus runtime) from Karavan. Providing the needful support in Karavan would make it easier to enforce these security measures.

**Comment by davsclaus** [comment, marked as answer]
This is not Karavan specific. The Rest DSL is from Apache Camel. So if you ask for some security configuration that you cannot find OOTB then I suggest you raise your ask at the core Apache Camel project
